luxonis_train/nodes/heads/bisenet_head.py

```diff
@@ -56,3 +57,13 @@ def forward(self, inputs: Tensor) -> Tensor:
         x = self.conv_3x3(inputs)
         x = self.conv_1x1(x)
         return self.upscale(x)
+
+    def get_custom_head_config(self) -> dict:
+        """Returns custom head configuration.
+
+        @rtype: dict
+        @return: Custom head configuration.
+        """
+        return {
+            "is_softmax": False,
+        }
```

Comment:
Segmentation parser in DAI-nodes needs "output_layer_name" and "classes_in_one_layer" - [reference](https://github.com/luxonis/depthai-nodes/blob/main/depthai_nodes/ml/parsers/segmentation.py)
(same for ddnet and segmentation head)